File: uri 1048.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    double os,ne,pe;
    int p;
    scanf("%lf",&os);
    if(os>=0.00 && os<=400.00){
        p=15;
        pe=os*0.15;
        ne=os+pe;
    }
    else if(os>=400.01 && os<=800.00){
        p=12;
        pe=os*0.12;
        ne=os+pe;
    }
    else if(os>=800.01 && os<=1200.00){
        p=10;
        pe=os*0.10;
        ne=os+pe;
    }
    else if(os>=1200.01 && os<=2000.00){
        p=7;
        pe=os*0.07;
        ne=os+pe;
    }
    else{
        p=4;
        pe=os*0.04;
        ne=os+pe;
    }
    printf("Novo salario: %.2lf\nReajuste ganho: %.2lf\nEm percentual: %d %%\n",ne,pe,p);
    return 0;
}

File: uri 1048.py
salary = float(input())

if 0 <= salary <= 400:
    new = salary * 1.15
    earned = salary * .15
    per = 15
elif 400 < salary <= 800:
    new = salary * 1.12
    earned = salary * .12
    per = 12
elif 800 < salary <= 1200:
    new = salary * 1.10
    earned = salary * .10
    per = 10
elif 1200 < salary <= 2000:
    new = salary * 1.07
    earned = salary * .07
    per = 7
else:
    new = salary * 1.04
    earned = salary * .04
    per = 4
print("Novo salario: %.2f"%new)
print("Reajuste ganho: %.2f"%earned)
print(f"Em percentual: {per} %")
